gmail/src/views/leads.ts

```diff
@@ -105,8 +108,12 @@ export function buildLeadsView(state: State, card: Card) {
                 ),
             );
         }
-    } else {
+    } else if (state.canCreatePartner) {
         leadsSection.addWidget(CardService.newTextParagraph().setText(_t("Save Contact to create new Opportunities.")));
+    } else {
+        leadsSection.addWidget(
+            CardService.newTextParagraph().setText(_t("The Contact needs to exist to create Opportunities.")),
```

Comment:
It is a bit strangely said ... Maybe something like "You can only create opportunities for existing customers" ?